nix/overlays.nix

```diff
@@ -0,0 +1,40 @@
+self: final: prev: {
+  pythonPackagesOverlays =
+    (prev.pythonPackagesOverlays or [])
+    ++ [
+      (_: pprev: {
+        pywlroots = (pprev.pywlroots.overrideAttrs(_: rec {
+          version = "0.17.0";
+          src = prev.fetchFromGitHub {
+            owner = "flacjacket";
+            repo = "pywlroots";
+            rev = "bf314bae5d7a9552225c44ab1e7bf7e4fafa869e";
+            hash = "sha256-tvem8gmih9EfseZ3MgU1UW05xfttzKLcJEaymM5uNPI=";
+          };
+        })).override {
+          wlroots = prev.wlroots_0_17;
+        };
+        qtile = (pprev.qtile.overrideAttrs (old: let
+          flakever = self.shortRev or "dev";
+        in {
+          version = "0.0.0+${flakever}.flake";
```

Comment:
We need a way to deduce the version number, maybe you can parse it from the changelog?

Reply:
I don't understand what you want me to do, can you explain better, what do you want me to parse from changelog?

Reply:
Now it says 0.0.0+commit.flake,

It should say 0.28.1.flake for a release buikd

Reply:
Ohh i get it, so the version + flake

Reply:
Yeah

Reply:
For dev/git commits that are not tagged yet 0.0.0 or so is fine